.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Please add a global .gitignore on your own device for your development process. As a project maintainer, I don't want to have to manage pre-project .gitignore for every potential IDE.

Reply:
Done thank you.